packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -58,6 +64,9 @@ class Carousel extends Foundation<
         items: void 0,
     };
 
+    private slideTransitionDirection: Partial<SlideTransitionDirection> =
```

Comment:
Are there things that might TransitionDirection other than slides?  It might be nice to extract this out one level.  Then have TransitionDirection or TransitionTiming etc that could be applied to many different things besides just slides.

Reply:
`Slide` is just the internal language of content blocks. Once the direction is known, the appropriate class is added. Then the user can add any animation or transition needed.